docs/reference/setup/install/docker.asciidoc

```diff
@@ -0,0 +1,290 @@
+[[docker]]
+=== Install Elasticsearch with Docker
+
+Elasticsearch is also available as a Docker image.
+The image is built with https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack].
+
+==== Security note
+
+NOTE: https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] is preinstalled in this image.
```

Comment:
Fixed in https://github.com/elastic/elasticsearch/pull/21497/commits/d7646bdbb166a0781130f558637da93214c3ec23